tsl/test/sql/vectorized_aggregation.sql

```diff
@@ -349,7 +349,7 @@ value2 AS segment_by_value2,
 value1 AS int_value,
 value1 AS float_value
 FROM
-generate_series('1980-01-01 00:00:00-00', '1980-03-01 00:00:00-00', INTERVAL '1 day') AS g1(time),
+generate_series('1980-01-03 00:00:00-00', '1980-03-04 00:00:00-00', INTERVAL '1 day') AS g1(time),
```

Comment:
Spent a lot of time here on a flaky plan -- turns out the Parallel Append sorts the children by cost, and one chunk was smaller. Made them same size to avoid flakiness.